### ISSUE TITLE
For October 2018

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#154]
## Summary

> _Specify the total amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ requested: `2400`
 - BSQ address: B19yxYCRTVaDfRfucrH6aEzeZpoVTv8Je8S

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule
- takes into account the time spent for the roles

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-434391209 | (BSQ  requested = 2000)
Evaluation is based and intended in line with : previous compensation requests, https://github.com/bisq-network/compensation/issues/24 , and other support compensation requests.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = 200)
TL;DR: there was one unique catch this month (nodes are running well).

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-434398889 | (BSQ requested = 200)

## Other points (BSQ requested = 0)
- Onion(s) in installation

## Contributions in progress
- regtest environment (hopefully will be usable for v.0.9.0)
- in addition to existing monitoring alerts, I'm seing for online daily monitoring/reporting.

**Comment by cbeams** [comment]
@HarryMacfinned, thanks for following the reporting formatting instructions I originally laid out at https://docs.bisq.network/roles.html#reporting. In practice, though, most of us (including myself) have been using a different, and I think more natural / readable format. I've updated the doc accordingly, please have a look at https://github.com/bisq-network/bisq-docs/pull/91 to see the changes.

**Comment by ripcurlx** [comment]
Closing as complete, see https://github.com/bisq-network/compensation/issues/149#issuecomment-435645489.
